modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -468,6 +516,36 @@ TEST_F(ONNXtest, InferMultOutput)
     // Validate
     validate();
 }
+
+TEST_F(ONNXWithRemap, InferWithDisabledOut)
+{
+    useModel("/object_detection_segmentation/faster-rcnn/model/FasterRCNN-10");
+    // Create tensor
+    cv::Mat rand_mat = initMatrixRandU(CV_32FC3, cv::Size{800, 800});
+    std::vector<int> dims = {rand_mat.channels(), rand_mat.rows, rand_mat.cols};
+    cv::Mat in_mat(dims, CV_32F, rand_mat.data);
+    // ONNX_API code
+    infer<float>(in_mat, out_onnx, {"6379", "6383"});
+    // G_API code
+    using FRCNNOUT = std::tuple<cv::GMat, cv::GMat>;
+    G_API_NET(FasterRCNN, <FRCNNOUT(cv::GMat)>, "FasterRCNN");
+    auto net = cv::gapi::onnx::Params<FasterRCNN>{model_path}
+        .cfgOutputLayers({"out1", "out2"})
+        .cfgPostProc({cv::GMatDesc{CV_32F, {1,20}},
+                      cv::GMatDesc{CV_32F, {5}}}, remapRcnnPorts)
+        .cfgDisableOutput("6381");
+
+    cv::GMat in, out1, out2;
+    std::tie(out1, out2) = cv::gapi::infer<FasterRCNN>(in);
+
+    cv::GComputation comp(cv::GIn(in), cv::GOut(out1, out2));
+    out_gapi.resize(2);
```

Comment:
You can use `num_out` here, I suppose? The string [:200](https://github.com/opencv/opencv/pull/18903/files#diff-81a0f32e0528f862f8ed8a7ba31d08fb9a21d8ab9d8fa90ec3a03fb75874b25eR200) is for that purpose?

Reply:
I can. Fixed.